server/tests/services/group.test.js

```diff
@@ -212,35 +212,38 @@ describe('# Group Service', () => {
           });
       });
 
-
-      it('should update a groups escalation policy', (done) => {
-        const newEscalationPolicy = {
-          pagingInterval: 1111,
-          rotationInterval: 2222
-        };
-
-        groupService.updateEscalationPolicy(group, newEscalationPolicy)
-          .then((savedGroup) => {
-            expect(savedGroup).to.exist;
-            expect(savedGroup.escalationPolicy.pagingInterval)
-              .to.equal(newEscalationPolicy.pagingInterval);
-            expect(savedGroup.escalationPolicy.rotationInterval)
-              .to.equal(newEscalationPolicy.rotationInterval);
-            done();
-          });
+      context('with correct body fields', () => {
+        it('should update a group\'s escalation policy', (done) => {
+          const newEscalationPolicy = {
+            pagingInterval: 1111,
+            rotationInterval: 2222
+          };
```

Comment:
This setup should be pulled out of the `it` block

    context('with correct body fields) 
      const newEscalationPolicy = {}
      it('should...